http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala

```diff
@@ -404,4 +413,80 @@ private[leonardo] object BuildHelmChartValues {
+                                             appName: AppName,
+                                             cloudContext: AzureCloudContext,
+                                             workspaceId: WorkspaceId,
+                                             landingZoneResources: LandingZoneResources,
+                                             relayPath: Uri,
+                                             petManagedIdentity: Option[Identity],
+                                             storageContainer: StorageContainerResponse,
+                                             batchAccountKey: BatchAccountKey,
+                                             applicationInsightsConnectionString: String,
+                                             sourceWorkspaceId: Option[WorkspaceId],
+                                             userAccessToken: String,
+                                             identityType: IdentityType,
+                                             maybeDatabaseNames: Option[CromwellRunnerDatabaseNames]
+  ): Values = {
+    val valuesList =
+      List(
+        // azure resources configs
+        raw"config.resourceGroup=${cloudContext.managedResourceGroupName.value}",
+        raw"config.batchAccountKey=${batchAccountKey.value}",
+        raw"config.batchAccountName=${landingZoneResources.batchAccountName.value}",
+        raw"config.batchNodesSubnetId=${landingZoneResources.batchNodesSubnetName.value}",
+        raw"config.drsUrl=${config.drsConfig.url}",
+        raw"config.landingZoneId=${landingZoneResources.landingZoneId}",
+        raw"config.subscriptionId=${cloudContext.subscriptionId.value}",
+        raw"config.region=${landingZoneResources.region}",
+        raw"config.applicationInsightsConnectionString=${applicationInsightsConnectionString}",
+
+        // relay configs
+        raw"relay.path=${relayPath.renderString}",
+
+        // persistence configs
+        raw"persistence.storageResourceGroup=${cloudContext.managedResourceGroupName.value}",
+        raw"persistence.storageAccount=${landingZoneResources.storageAccountName.value}",
+        raw"persistence.blobContainer=${storageContainer.name.value}",
+        raw"persistence.leoAppInstanceName=${appName.value}",
+        raw"persistence.workspaceManager.url=${config.wsmConfig.uri.renderString}",
+        raw"persistence.workspaceManager.workspaceId=${workspaceId.value}",
+        raw"persistence.workspaceManager.containerResourceId=${storageContainer.resourceId.value.toString}",
+
+        // identity configs
+        raw"identity.enabled=${identityType == PodIdentity}",
+        raw"identity.name=${petManagedIdentity.map(_.name).getOrElse("none")}",
+        raw"identity.resourceId=${petManagedIdentity.map(_.id).getOrElse("none")}",
+        raw"identity.clientId=${petManagedIdentity.map(_.clientId).getOrElse("none")}",
+        raw"workloadIdentity.enabled=${identityType == WorkloadIdentity}",
```

Comment:
I don't think we need these for cromwell-runner 🤔

Reply:
Hmm I thought that we do want to create Cromwell Runner app with Workload Identity but scoped to the user. @cjllanwarne Is that right?

Reply:
Yes, the `identity.xyz` configs are all related to the Pod Identity if I'm not mistaken, whereas we only care about the `workloadIdentity.serviceAccountName` since Cromwell Runner should only be created with workload identity and never with Pod

Reply:
Agree about just using workload identity -- and only passing helm values the chart actually uses.